django/src/rdwatch/utils/worldview_processed/stac_search.py

```diff
@@ -78,4 +78,5 @@ def worldview_search(
         headers={'x-api-key': environ['RDWATCH_SMART_STAC_KEY']},
     )
     with urlopen(request) as resp:
-        return json.loads(resp.read())
+        data = resp.read()
+        return json.loads(data)
```

Comment:
In the interest of minimizing redundant diffs, can this change be reverted? (unless it's intentional; I'm guessing it was broken out into a variable to assist in debugging though?)

Reply:
Yeah that was some testing stuff that can be removed, thanks for catching that.